src/dbcp/transform/google_maps.py

```diff
@@ -202,3 +214,61 @@ def _get_geocode_response(
         return response[0]
     except IndexError:  # empty list = not found
         return {}
+
+
+def _geocode_row(
```

Comment:
I moved this logic from `dbcp.transform.helpers` because it is specific to the Google Maps API.